README.rst

```diff
@@ -424,23 +341,30 @@ Consuming the Primary API URL (e.g. "https://connect.squareup.com/v2/")
         }
     }
 
-Root Component
-``````````````
-Unlike Guice and Dagger1, there is no ObjectGraph/Injector object that one can pull arbitrary instances out of.
-
-Cleanse has a concept of a ``Component``. A ``Component`` is essentially a ``Module``, but with an `associated type`_
-named ``Root``. The ``Root`` associated type in a component is the *Root* of the object graph. An instance of ``Root``
-is what's returned when a ``Component`` is constructed. It also may be referred to as an "entry point",
+Components
+`````````````````````
+Cleanse has a concept of a ``Component``. A ``Component`` represents an object graph of our dependencies that returns the ``Root`` `associated type`_ upon construction and is used as the "entry point" into Cleanse. However, we can also use a ``Component`` to create a subgraph inside our parent object graph, called a subcomponent. Subcomponents are closely related to scopes_ and are used to scope your dependencies. Objects inside a component are only allowed to inject dependencies that exist within the same component (or scope), or an ancestor's component. A parent component is not allowed to reach into a subcomponent and retrieve a dependency. One example of using components to scope dependencies is by having a ``LoggedInComponent`` inherting from your application's Root component. This allows you to bind logged in specific objects such as session tokens or account objects within the ``LoggedInComponent`` so that you can't accidently leak these dependencies into objects used outside of a logged session (i.e welcome flow views).
```

Comment:
`Subcomponents are closely related to scopes_ and are used to scope` what's with the `scopes_`/ Supposed to be link?